src/main/groovy/life/qbic/utils/NanoporeParser.groovy

```diff
@@ -122,19 +123,19 @@ class NanoporeParser {
 
     private static void checkPresenceOfFlowCellPosition(Map metadata, String flowCellEntry) {
         if (!metadata.containsKey(flowCellEntry)) {
-            throw new MissingArgumentException("Could not find metadata information about the flow cell position.")
+            throw new RuntimeException("Could not find metadata information about the flow cell position.")
```

Comment:
What was the reason for exchanging the exception type here 
I'd expect a missing argument to throw a MissingArgumentException?